Packages/com.unity.multiplayer.samples.coop/CHANGELOG.md

```diff
@@ -1,5 +1,110 @@
 # Multiplayer Samples Co-op Changelog
 
+## [0.2.0] - 2021-05-18
+
+v0.2.0 is an Early Access release for Multiplayer Samples Co-op.
+
+It requires and supports Unity v2020.3 and later and Unity MLAPI v0.1.0. For additional information on MLAPI, see the changelog and release notes for the Unity MLAPI package.
+
+### New features
+
+* Updated the user interface including the following:
+
+  * When joining a game, a "Connecting..." UI loads. When disconnecting from a game, you are returned to the MainMenuScene with a "Connection to Host lost" message. If the game fails to connect, a general message "Connection to Host failed" loads. 
+  * Added an option to leave the lobby after joining it. You no longer have to restart the game to leave the lobby.
+  * Added option to cancel the game after clicking **Ready**. 
+  * Added a gear icon for accessing and modifying app settings. 
+  * The UI now tracks and displays player has arrived, number of connected players in the lobby, and player status. 
+
+* Added F/X and animation assets for the game including:
+
+  * Audio files for boss sound effects
+  * Ramp-up animation for hero movement and actions
+  * Visual effects and particles for Tank charge skill 
+  * Art assets to wave spawner, including animations for ReceiveDamage, Broken (died), and Revive 
+  * Game music 
+
+* Added audiomixer to Boss Room for separate audio channels and a master mixer. 
+
+* Updated and added various hero abilities:
+
+  * Added a cooldown to Archer's PowerShot. 
+  * Used generic actions to implement Rogue's Dagger skill. 
+  * Used generic actions to implement Rogue's Sneak skill using local stealth, applying a graphical effect to the stealthy character while still sending all network events normally. 
+  * Used generic actions to implement Rogue's Dash skill. 
+  * Properly display Heal abilities when targeting a fallen ally character. 
+  * Character attack actions properly support Hold to charge options. 
+
+* To show how UI elements and game objects can be networked, added networked functionality using `INetworkSerializable` for UI elements on the Character Selection screen including networked mouse cursors. 
+* Added a Photon filter for host room names not to allow profanity, racial slurs, and additional socially acceptable words. 
```

Comment:
Just confirming, are names actually filtered for profanity? I know we had a task to introduce sanitization but that was mostly to filter out invisible characters introduced by Zoom when copy-pasting room names.

Reply:
Gotcha, the ticket was thin on details and didn't easily find into in PRs. Updating based on this. :)

Reply:
Just checking. It seems this change remained.